src/test/java/net/snowflake/client/jdbc/telemetryOOB/TelemetryServiceTest.java

```diff
@@ -0,0 +1,77 @@
+package net.snowflake.client.jdbc.telemetryOOB;
```

Comment:
I like the new tests, but is there a reason for creating a new test file? I think these could probably be added to TelemetryServiceIT.java. Or if there is a compelling reason to create a new test file, it should have the "IT" on the end so it's easily identifiable as a test file.

Reply:
Oh yeah, because these new added functions don't need to run with connection. They are just unit tests. I found our existed unit test files are all end up with `xxxTest.java`, while our integration tests are usually named as `xxxIT.java`. I believe this name pattern is used by maven plugin.

Reply:
The unit tests run as part of the build so that you can catch the regression earlier. If the tests are runnable without connection, they should be in the unit tests.